Probesys/bileto
# new: Add observers to a tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/39

## How to test manually



1. create a ticket with a requester and an assignee (both should be different)
2. once created, add an observer to the ticket by editing the actors (the observer must not have the permission to see all the tickets of the organization)
3. verify that the observer appears in the list of actors
4. add an answer with the requester or the assignee
5. verify that a mail notification is sent to the other users (including the observer)
6. login with the observer and go to the list of tickets
7. verify that you see the ticket and that you can open it
8. go back to the list of tickets and open the advanced search form
9. search for `observer:@me`
10. verify that you see only the ticket you're observing

## Checklist
